scripts/sharedFuncs.sh

```diff
@@ -192,12 +193,12 @@ function download_component() {
                     notify-send "Photoshop CC" "$4 download completed" -i "download"
                 fi
 
-            elif [ "$curlpkg" == "true" ];then
-                show_message "using curl to download $4"
-                curl $3 -o $1
-            else
+            elif [ "$wgetpkg" == "true" ];then
                 show_message "using wget to download $4"
-                wget --no-check-certificate "$3" -P "$CACHE_PATH"
+                wget --no-check-certificate "$3" --continue --tries=0 -P "$CACHE_PATH"
+            else
```

Comment:
and in else block we can print error with sugestion to install aria/wget and retry